src/app/Rules/Rules.tsx

```diff
@@ -236,7 +289,20 @@ export const Rules = () => {
           <ToolbarContent>
             <ToolbarGroup variant="icon-button-group">
               <ToolbarItem>
-                <Button key="create" variant="primary" onClick={handleCreateRule}>Create</Button>
+                <Dropdown 
+                  onSelect= {()=>setIsDropdownOpen(false)} 
+                  toggle={
+                    <DropdownToggle
+                      toggleIndicator={null}
+                      onToggle={onRuleGenerationMenuToggle} 
+                      id="toggle-rule-generate-action">
+                    <PlusIcon/>
```

Comment:
Yeh sure originally I thought they are pretty much the same. I could change to follow this pattern instead.

Reply:
![Screenshot from 2022-06-13 16-05-11](https://user-images.githubusercontent.com/68053619/173435918-006cdd01-52c3-47ab-a87f-b39573e0c56f.png)

@andrewazores something like this?

Reply:
Yea, that's good too. The `<UploadIcon />` in the example link I shared seems neat too, could be worth experimenting with that instead of the `Upload` text button.

We do now have an "upload" action in the Automated Rules, Archives, and Events (Templates) views, and all three use a different style for the action/button. They should be uniform. I think my preference is to use the upload icon in all of these cases.

Reply:
Yeh I misread the your comments. How about now? Should it has some tooltips or do you think the button itself is already self-explantory?

![Screenshot from 2022-06-13 16-24-24](https://user-images.githubusercontent.com/68053619/173438926-9002ee85-c6b4-4e5f-a3e8-7a201fe5823a.png)

Reply:
I put a sample secondary icon button. Is there a specific style you want the button to be @andrewazores?

Reply:
I think that looks good, even without the tooltip. Ideally the icon alone makes enough sense for the user that they don't need an additional explanation for what it does.

Reply:
Could you update the "Upload" buttons on the other two views to match this one?

Reply:
Yeh sure I will get that done

Reply:
For Archive view:

![Screenshot from 2022-06-13 21-22-37](https://user-images.githubusercontent.com/68053619/173474919-e9112b92-4ea0-4b74-9a42-38354e30b80f.png)

For Events view:

![Screenshot from 2022-06-13 21-32-42](https://user-images.githubusercontent.com/68053619/173474927-df0a50da-612c-4e82-baad-8ac9b88da7b2.png)